local/snapshot.go

```diff
@@ -122,20 +122,22 @@ func (ln *localNetwork) SaveSnapshot(ctx context.Context, snapshotName string) (
 		return "", err
 	}
 	// create main snapshot dirs
-	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDbSubdir))
+	// TODO for CLI team -- what is the line below supposed to do?
```

Comment:
@felipemadero the first `Join` isn't doing anything, that's why the linter complains. What was the idea?

Reply:
ohh this is just an error, just remove the outer join